RELEASE/scripts/excavator/x_utils.ash

```diff
@@ -67,4 +69,35 @@ void call_registered_projects( string event, string meta, string page )
     {
         call void function_name( meta, page );
     }
+}
+
+void send_spading_data()
+{
+    string spading_data = get_property( "spadingData" );
+
+    string [int] pieces = spading_data.split_string( "|" );
+
+    int i = 0;
+    while ( i < count(pieces) - 4 )
+    {
+        string contents = pieces[i];
+        i += 2;
+        string recipient = pieces[i];
+        i += 2;
+        string explanation = pieces[i];
+        i += 2;
+
+        kmail(recipient, contents, 0);
```

Comment:
A regular expression might be more efficient for this. I could write one, if you agree.